UI/cmake/os-windows.cmake

```diff
@@ -0,0 +1,47 @@
+if(NOT TARGET OBS::blake2)
+  add_subdirectory("${CMAKE_SOURCE_DIR}/deps/blake2" "${CMAKE_BINARY_DIR}/deps/blake2")
+endif()
+
+if(NOT TARGET OBS::w32-pthreads)
+  add_subdirectory("${CMAKE_SOURCE_DIR}/deps/w32-pthreads" "${CMAKE_BINARY_DIR}/deps/w32-pthreads")
+endif()
+
+find_package(MbedTLS)
+
+configure_file(cmake/windows/obs.rc.in obs.rc)
+
+target_sources(
+  obs-studio
+  PRIVATE cmake/windows/obs.manifest
+          platform-windows.cpp
+          obs.rc
+          update/update-window.cpp
+          update/update-window.hpp
+          update/win-update.cpp
+          update/win-update.hpp
+          update/shared-update.cpp
+          update/shared-update.hpp
```

Comment:
Those are added in `feature-whatsnew.cmake` because of how the entire update system is now tied into having browser panels, the "What's New" system, and our proprietary communication with the server.

That change came after I wrote the rework, so I had to kinda-sorta patch it in (e.g. Sparkle is only added if the "What's New" feature is also enabled because of the whole "update channel" thing).

Reply:
As long as all the files are properly included for all use-cases, that's fine.